apps/wistia-videos/src/components/Field.tsx

```diff
@@ -115,30 +143,43 @@ const Field = (props: FieldProps) => {
                   placeholder="Search for a video"
                 />
               </Flex>
-              <Flex style={{ width: '100%', height: '380px', overflow: 'scroll' }}>
-                <Grid columns={3} columnGap={'spacingS'} rowGap={'spacingS'}>
-                  {[...dropdownData].map((item) => (
-                    <GridItem key={item.id}>
-                      <Card
-                        onClick={() => updateVideoIds(item.id)}
-                        style={{ height: '100px', padding: '7px' }}
-                        selected={selectedIds.findIndex((id) => item.id === id) !== -1}
-                      >
-                        <StyledImageContainer>
-                          <img src={item.thumbnail.url} alt={item.name} />
-                        </StyledImageContainer>
-                      </Card>
-                      <Paragraph>{item.name}</Paragraph>
-                    </GridItem>
-                  ))}
-                </Grid>
+              <Flex
```

Comment:
nit: If you made those state changes in the comments above , on line 138 you can now just make this 
```
data.length
```

Reply:
Resolving this beacause it's the same comment on line 140